cmd/argocd/commands/app.go

```diff
@@ -1040,6 +1040,8 @@ func NewApplicationDiffCommand(clientOpts *argocdclient.ClientOptions) *cobra.Co
 func NewApplicationDeleteCommand(clientOpts *argocdclient.ClientOptions) *cobra.Command {
 	var (
 		cascade bool
+		wait    bool
+		timeout int
 	)
 	var command = &cobra.Command{
```

Comment:
We're trying to complete command examples when they are missing. Can you please check and add a couple of usage example consistent with other commands?

Reply:
Will do!

Reply:
done